apps/site/pages/en/about/index.mdx

```diff
@@ -77,3 +77,44 @@ by using our [`child_process.fork()`](https://nodejs.org/api/child_process.html)
 communicate with. Built upon that same interface is the [`cluster`](https://nodejs.org/api/cluster.html) module,
 which allows you to share sockets between processes to enable load balancing
 over your cores.
+
+## Official Node.js Resources
+
+To ensure authenticity and security when working with Node.js, always use official sources. Avoid trusting emails,
+binaries, or downloads from unofficial sources.
+
+### Official Node.js Domains
+
+For downloading Node.js binaries and accessing official documentation, use only these domains:
+
+- [nodejs.org](https://nodejs.org)
+- [nodejs.dev](https://nodejs.dev)
```

Comment:
```suggestion
- [nodejs.dev](https://nodejs.dev) - redirecting to `nodejs.org`, if you have a link to this domain please update it to nodejs.org
```

Reply:
To be fair, all of the domains listed redirect to nodejs.org. I'll add a comment

Reply:
okay I wasn't aware for iojs